core/jobs/transforms/validation/base_validation_test.py

```diff
@@ -259,51 +275,105 @@ def test_validate_post_commit_is_public_raise_exception(self):
 
 
 class MockValidateModelDomainObjectInstancesWithNeutral(
-        base_validation.ValidateModelDomainObjectInstances):
-    def _get_model_domain_object_instance(self, item): # pylint: disable=unused-argument
+    base_validation.ValidateModelDomainObjectInstances[base_models.BaseModel]
+):
+    def _get_model_domain_object_instance(
+        self, item: base_models.BaseModel  # pylint: disable=unused-argument
```

Comment:
```suggestion
        self, _: base_models.BaseModel
```

Reply:
I think we can take away this ignore by replacing the parameter name with '_'

DItto elsewhere

Reply:
Done wherever it is applicable!!